02/app.js

```diff
@@ -1 +1,8 @@
-console.log('DOM');
\ No newline at end of file
+const dataElements = document.querySelectorAll('[data-url]');
+if (dataElements) {
```

Comment:
Tutaj `if` już nie jest niezbędny. Przy wyszukiwaniu wielu elementów mamy tablicę - pustą lub nie.
Taka tablica (pusta lub nie) zawsze jest prawdą i ten warunek nic nie daje.

Jak bardzo chcemy mieć warunek (a nie jest on potrzebny bo przy pustej tablicy po prostu `for` się nie wykona) to trzeba sprawdzić długość tablicy tj. `if(dataElements.length)`